backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -41,6 +47,25 @@
     private BacktraceDatabaseSettings databaseSettings;
     private boolean _enable = false;
 
+    /**
+     * Add attributes to native reports
+     * @param name attribute name
+     * @param value attribute value
+     */
+    public native void AddAttribute(String name, String value);
+
+    /**
+     * Initialize Backtrace-native integration
+     * @param url url to Backtrace
+     * @param databasePath path to Backtrace-native database
+     * @param handlerPath path to error handler
+     * @param attributeKeys array of attribute keys
+     * @param attributeValues array of attribute values
+     * @return true - if backtrace-native was able to initialize correctly, otherwise false.
+     */
+    private native boolean Initialize(String url,  String databasePath, String handlerPath, String[] attributeKeys, String[] attributeValues);
```

Comment:
in Java it is normal to start the name of a method from a lower case letter

Reply:
renamed to lowercase - other methods as well.